app/views/assignments/index.html.erb

```diff
@@ -17,7 +17,7 @@
 
 <% if @assignments.empty? %>
   <p class="none_created">No Assignment has been created.</p>
-<% elsif @guest_user %>
+<% elsif @guest_details.length > 0 %>
```

Comment:
Here, guest_details.length is later in your view described as "processed assignments", so does this mean the details are "processed assignments" (as opposed to just assignments)?